src/scancode/api.py

```diff
@@ -142,6 +142,7 @@ def get_licenses(
     license_url_template=SCANCODE_LICENSEDB_URL,
     unknown_licenses=False,
     deadline=sys.maxsize,
+    additional_directories=None,
```

Comment:
IMHO this is not needed. As explained earlier, I think that considering additional directories should  be something that's only done when re-indexing and not when scanning. Asking for additional_directories should be an error when scanning.
So much so that I would think as the `--reindex-licenses` option pretty much as if it were a separate command entirely.